ethereum/core/src/main/java/org/hyperledger/besu/ethereum/trie/common/AbstractStateTrieAccountValue.java

```diff
@@ -76,6 +78,16 @@ public Hash getStorageRoot() {
     return Hash.EMPTY_TRIE_HASH;
   }
 
+  /**
+   * The size of the EVM bytecode associated with this account.
+   *
+   * @return the size of the account code (which may be {@link Optional#empty()}).
+   */
+  @Override
+  public Optional<Long> getCodeSize() {
+    return Optional.empty();
+  }
```

Comment:
how can `codeHash != Hash.EMPTY` and code size be empty? It doesn't seem to make sense

Reply:
Optional.empty is not == 0 it's a way to say that the codesize is not available but I modified the code and this method is now abstract  and in **PmtStateTrieAccountValue** I throw an exception 

```java 
@Override
  public Optional<Long> getCodeSize() {
    throw new UnsupportedOperationException("Retrieving code size from state trie account value is not possible with Patricia Merkle Trie");
  }
```